client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -22,6 +22,41 @@ define([
     ];
     function PostsService(api, $q, userList) {
 
+        /**
+         * Fetch a page of posts
+         * @param {string} blog_id - The id of the blog
+         * @param {object} filters - (available: {boolean} 'status', {string} 'updatedAfter')
+         * @param {integer} max_results - maximum number of results per page
+         * @param {integer} page - page index
+         */
+        function getPosts(blog_id, filters, max_results, page) {
+            filters       = filters     || {};
+            page          = page        || 1;
+            max_results   = max_results || 15;
+            var posts_criteria = {
+                source: {
+                    query: {filtered: {filter: {and: [{not: {term: {deleted: true}}}]}}}
+                },
+                page: page,
+                max_results: max_results
+            };
+            // filters.status
+            if (angular.isDefined(filters.status)) {
+                posts_criteria.source.query.filtered.filter.and.push({term: {post_status: filters.status}});
+            }
+            // filters.updatedAfter
+            if (angular.isDefined(filters.updatedAfter)) {
+                posts_criteria.source.query.filtered.filter.and.push({
```

Comment:
Can you tell why? I think it's good to have default values because it's a service, and service methods should be as simple as possible